src/poliastro/bodies.py

```diff
@@ -83,11 +83,22 @@ def __init__(self, parent, k, name, symbol=None, R=0 * u.km):
         self.name = name
         self.symbol = symbol
         self.R = R
+        self.kwargs = kwargs
 
     @classmethod
     @u.quantity_input(k=u.km ** 3 / u.s ** 2, R=u.km)
-    def from_parameters(cls, parent, k, name, symbol, R):
-        return cls(parent, k, name, symbol, R)
+    def from_parameters(cls, parent, k, name, symbol, R, **kwargs):
+        return cls(parent, k, name, symbol, R, **kwargs)
+
+    @classmethod
+    def from_relative(cls, reference, parent=None, k=None, name=None,
+                      symbol=None, R=None, **kwargs):
+        if hasattr(reference, 'k') and hasattr(reference, 'R'):
+            k = k * reference.k
+            R *= R * reference.R
```

Comment:
Isn't this `R = R * reference.R` or `R *= reference.R`? In any case be consistent with `k` (unless I have misunderstood something).